internal/msgstream/result

```diff
@@ -0,0 +1,63 @@
+=== RUN   TestStream_PulsarMsgStream_3
```

Comment:
what does this file do?

Reply:
It is used to test mqMsgStream's seek function(different from mqTtMsgStream).

Reply:
I have made changes in this test's name.